### DISCUSSION TITLE
Textbox can support text-indent？

### DISCUSSION THREAD

**Discussion by Chenirving** [fabricjs/fabric.js#9631] [Q&A]
I wanted to indent the first line, but I didn't find the answer I was looking for in the documentation, demo?
<img width="776" alt="Xnip2024-01-24_15-20-03" src="https://github.com/fabricjs/fabric.js/assets/14194443/aa7fd6d5-cb75-4bdf-b92c-3ce4b5a290ee">

**Comment by asturur** [comment]
it supports only through override as of today
Are you trying to add a bullet list?
